src/api/spec/models/attrib_spec.rb

```diff
@@ -76,4 +76,31 @@
       it { expect(attribute_with_package.project).to eq(package.project) }
     end
   end
+
+  context "#update_with_associations" do
+    context "without issues and without values" do
+      it { expect(attribute.update_with_associations).to be false }
+
+      context "add an issue" do
```

Comment:
What about a test when you already have an issue / value (that it gets overwritten)?